<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>751</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane { /* The tab widget frame */
	background-image: url(:/back/back.png);

	color: rgb(98, 213, 255);


border-top: 5px solid #535353;
}
QTabWidget::tab-bar {
left: 0px; /* move to the right by 5px */
}
/* Style the tab using the tab sub-control. Note that it reads QTabBar not QTabWidget */
QTabBar::tab {
background: #dcdcdc;
border: 2px solid #535353;
border-bottom-color: #535353; /* same as the pane color */
border-top-left-radius: 7px;
border-top-right-radius: 7px;
min-width: 120px;
padding: 8px;
color: #6a1616;
font-size:15px;
font-family:Arial, Helvetica, sans-serif;
}
QTabBar::tab:selected, QTabBar::tab:hover {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #c3a1a1, stop: 0.4 #62d5ff, stop: 0.5 #439eff, stop: 1.0 #3a65ff);
}
QTabBar::tab:selected {
border-color: #9B9B9B;
border-bottom-color: #535353; /* same as pane color */
color: #dcdcdc;
font-size:15px;
font-family:Arial, Helvetica, sans-serif;
}
QTabBar::tab:!selected {
margin-top: 2px; /* make non-selected tabs look smaller */
}
QLineEdit {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #ccc;
  border-radius: 4px;
  box-sizing: border-box;
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>famille</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-image: url(:/back/images.png);</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_25">
       <attribute name="title">
        <string>ajouter</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom de famille</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Pr√©nom de reponsable</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>221</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>221</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>170</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>50</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>email responsable</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="email">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>30</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>120</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>password</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>190</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>pays </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>100</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>169</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="ajoutparents">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>270</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="modifierparents">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>270</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier </string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>affichage</string>
       </attribute>
       <widget class="QTableView" name="tableaffiche">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>481</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="recherche_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>230</y>
          <width>321</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="recherche">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>recherche</string>
        </property>
       </widget>
       <widget class="QPushButton" name="trier">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>120</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>trier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pdf">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>50</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>suppression</string>
       </attribute>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>120</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_id_sup">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>90</y>
          <width>241</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supprimer">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>110</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>mailing</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_12">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>811</width>
          <height>431</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/pic/pic.jpeg);</string>
        </property>
        <property name="title">
         <string>E-mailing</string>
        </property>
        <widget class="QLineEdit" name="subject_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>80</y>
           <width>581</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">
padding: 0 8px;



border-width:2px;
border-style:solid;
border-color:red;
background: #d1d1d1;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_Mail_2">
         <property name="geometry">
          <rect>
           <x>660</x>
           <y>350</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;
border-radius: 10px;
padding: 0 8px;
background:#FFB6C1	;
font: bold, &quot;Comic Sans MS&quot;;</string>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_mail_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>581</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width:2px;
border-style:solid;
border-color:red;
background: #d1d1d1;</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="msg_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>150</y>
           <width>581</width>
           <height>201</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">
padding: 0 8px;



border-width:2px;
border-style:solid;
border-color:red;
background: #d1d1d1;</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>enfant</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/back/images.png);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>ajouter</string>
       </attribute>
       <widget class="QLineEdit" name="id_line">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>20</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="linenom">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>70</y>
          <width>201</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineprenom">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>130</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="nom">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>90</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>nom de l'enfants </string>
        </property>
       </widget>
       <widget class="QLabel" name="id">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>40</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>ID </string>
        </property>
       </widget>
       <widget class="QLabel" name="prenom">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>150</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>prenom de l'enfants </string>
        </property>
       </widget>
       <widget class="QPushButton" name="quitter">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter_enfants">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>220</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>ajouter </string>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier_enfants">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>220</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>affichage</string>
       </attribute>
       <widget class="QTableView" name="tableaffichage_enfants">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>491</width>
          <height>231</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="trier_2">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>30</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>trier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pdf_2">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>70</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
       <widget class="QPushButton" name="recherche_3">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>180</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>recherche</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="idrecherche">
        <property name="geometry">
         <rect>
          <x>522</x>
          <y>109</y>
          <width>221</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>supprimer</string>
       </attribute>
       <widget class="QLineEdit" name="line_ID_sup_enf">
        <property name="geometry">
         <rect>
          <x>162</x>
          <y>90</y>
          <width>251</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>ID supp</string>
        </property>
       </widget>
       <widget class="QPushButton" name="suppression_enf">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>110</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
